src/commons/api/roas.rs

```diff
@@ -129,29 +129,35 @@ impl RoaAggregateKeyFmtError {
     }
 }
 
-//------------ RoaDefinition -----------------------------------------------
+//------------ RoaPayload --------------------------------------------------
 
-/// This type defines the definition of a Route Origin Authorization (ROA), i.e.
-/// the originating asn, IPv4 or IPv6 prefix, and optionally a max length.
+/// This type defines the definition of a Route Origin Authorization (ROA)
+/// payload: ASN, Prefix and optional Max Length
+///
+/// Note that a ROA object may contain multiple payloads (RoaDefinitions)
+/// aggregated by (the same) ASN. This is discouraged and Krill prefers
+/// to generate a dedicated ROA object for each payload, but aggregation
```

Comment:
What is this "ROA object" that is referred to in multiple places? Is there a suitable RFC link we can make to this or is it a thing in Krill or...?

Reply:
fixed - I did not use a link but I mention the RFC number at least so it can be found easily.